libs/cimit-service/src/test/java/uk/gov/di/ipv/core/library/pact/ContractTest.java

```diff
@@ -0,0 +1,751 @@
+        // Arrange
+        when(mockConfigService.getParameter(CIMIT_API_BASE_URL))
+                .thenReturn(getMockApiBaseUrl(mockServer));
+
+        var testVc =
+                VerifiableCredential.fromValidJwt(
+                        MOCK_USER_ID, null, SignedJWT.parse(FAILED_DVLA_VC_WITH_CI_JWT));
+
+        var underTest = new CiMitService(mockConfigService);
+
+        // Act
+        assertDoesNotThrow(() -> underTest.submitVC(testVc, MOCK_GOVUK_SIGNIN_ID, MOCK_IP_ADDRESS));
+    }
+
+    @Pact(provider = "CiMitProvider", consumer = "IpvCoreBack")
+    public RequestResponsePact postCiInvalidIssuerReturns400(PactDslWithProvider builder) {
+        var response = getFailedApiResponse("BAD_VC_ISSUER");
+
+        return builder.given("mockIpAddress is the ip-address")
+                .given("mockGovukSigninJourneyId is the govuk-signin-journey-id")
+                .given("mockUserId is the user_id")
+                .given("invalidIssuer is the issuer")
+                .given("the VC is from DCMAW-5477-AC1")
+                .given("the VC has CI code TEST-CI-CODE")
+                .uponReceiving("Request with invalid issuer")
+                .method("POST")
+                .path(POST_CI_ENDPOINT)
+                .body(
+                        String.format(
+                                "{\"signed_jwt\": \"%s\"}", DVLA_VC_WITH_CI_AND_INVALID_ISSUER_JWT))
+                .headers(
+                        IP_ADDRESS_HEADER,
+                        MOCK_IP_ADDRESS,
+                        GOVUK_SIGNIN_JOURNEY_ID_HEADER,
+                        MOCK_GOVUK_SIGNIN_ID,
+                        PactDslRequestBase.CONTENT_TYPE,
+                        "application/json")
+                .willRespondWith()
+                .body(response)
+                .status(400)
+                .toPact();
+    }
+
+    @Test
+    @PactTestFor(pactMethod = "postCiInvalidIssuerReturns400")
+    void failsToPostCis_whenCalledWithInvalidIssuerAgainstCimiApi_returns400(MockServer mockServer)
+            throws ParseException, CredentialParseException {
+        // Arrange
+        when(mockConfigService.getParameter(CIMIT_API_BASE_URL))
+                .thenReturn(getMockApiBaseUrl(mockServer));
+
+        var testVc =
+                spy(
+                        VerifiableCredential.fromValidJwt(
+                                MOCK_USER_ID,
+                                null,
+                                SignedJWT.parse(DVLA_VC_WITH_CI_AND_INVALID_ISSUER_JWT)));
+
+        var underTest = new CiMitService(mockConfigService);
+
+        // Act/Assert
+        assertThrows(
+                CiPutException.class,
+                () -> underTest.submitVC(testVc, MOCK_GOVUK_SIGNIN_ID, MOCK_IP_ADDRESS),
+                FAILED_API_REQUEST);
+    }
+
+    @Pact(provider = "CiMitProvider", consumer = "IpvCoreBack")
+    public RequestResponsePact successfullyReceivesMitigations(PactDslWithProvider builder) {
```

Comment:
Naming of this method could be more similar to the others e.g.: `postMitigationsSuccessfullyPostsMitigations` to mirror `postCiSuccessfullyPostsContraIndicator`